### ISSUE TITLE
Uncompatible with WorldGuard 1.16.1

### ISSUE THREAD

**Issue by DonMicha** [KernelFreeze/MinersTorch#2]
The plugin seems to have issues when giving permission to VIPS or any default player, they're able to place torches in any protected region, even when changing the flags.

**Comment by dgfonseca** [comment]
I agree with DonMicha, there are issues with different protection plugins such as World Guard and Protection stones, they bypass the protected areas.

**Comment by KernelFreeze** [comment]
Thank you for your report. I will check for possible solutions.

**Comment by KernelFreeze** [comment]
Does the latest commit work for you?
